core/declarations.gms

```diff
@@ -185,6 +185,10 @@ $ifthen.VREPot_Factor not "%c_VREPot_Factor%" == "off"
   p_VREPot_Factor(all_te) "Rescale factor for renewable potentials" / %c_VREPot_Factor% /
 $endif.VREPot_Factor
 
+$ifthen.scaleDemand not "%cm_scaleDemand%" == "off"
+  pm_scaleDemand(ttot,ttot,all_regi)   "Rescaling factor on final energy and usable energy demand, for selected regions and over a phase-in window." / %cm_scaleDemand% /
```

Comment:
You could consider defining the parameter over `tall` for users that want more precise trajectories of the rescaling factor. Of course, you would then have to use the aliases `tall2` and `tall3` in `code/datainput.gms`.

Reply:
good idea thanks